src/index.html

```diff
@@ -1,13 +1,338 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>BOSE</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="/" class="logo">
+              <img
+                class="logo__image"
+                src="/images/logo.png"
+                alt="The world Your music shines"
+              >
+            </a>
+            <div class="header__block-1">
+              <a
+               href="tel:+1 234 5555-55-55"
+               class="icon icon--phone header__call"
+              >
+              </a>
+              <a href="#menu" class="icon icon--menu">
+              </a>
+            </div>
+          </div>
+          <h1 class="header__title">
+            The world shades. Your music shines.
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
```

Comment:
It's weird that I click on menu on the right side of the page and it comes out from the left side

Reply:
Checklist 18th item
![image](https://user-images.githubusercontent.com/37161315/170694785-e7189025-f169-489c-a12b-18450d83d799.png)